src/videojs-contrib-hls.js

```diff
@@ -87,6 +87,9 @@ export default class HlsHandler extends Component {
     // HlsSourceHandler.handleSource()
     // @link https://github.com/videojs/video.js/pull/3245
     this.options_ = videojs.mergeOptions(options, videojs.options.hls || {});
```

Comment:
well object level options don't really exist atm, unless that PR I mentioned was recently merged.